core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1534,7 +1534,10 @@ setting.info.msg=When selling BTC for XMR you can use the auto-confirm feature t
   You can also set the maximum amount of BTC per trade to auto-confirm as well as the number of required \
   confirmations here in Settings.\n\n\
   See more details (including how to set up your own explorer node) on the Bisq wiki \
-  [HYPERLINK:https://bisq.wiki/Trading_Monero#Auto-confirming_trades]
+  [HYPERLINK:https://bisq.wiki/Trading_Monero#Auto-confirming_trades]\n\n\
+  Please be aware that when changing the Bisq data directory you must change the Monero wallet and address to avoid \
+  vulnerabilities related to transaction key re-use. More details can be found here: [HYPERLINK:https://web.getmonero.org/2018/09/25/a-post-mortum-of-the-burning-bug.html].
```

Comment:
```suggestion
  vulnerabilities related to transaction key re-use. More details can be found here: [HYPERLINK:https://www.getmonero.org/2018/09/25/a-post-mortum-of-the-burning-bug.html].
```
Super minor, but please use `www`, which is the canonical address. Using `web` would result in a cache skip.